hal/generic/TimerServiceGeneric.cpp

```diff
@@ -49,11 +50,10 @@ namespace hal
                 nextTrigger = infra::TimePoint::max();
 
                 infra::EventDispatcher::Instance().Schedule([this]()
-                {
+                    {
                     std::unique_lock<std::recursive_mutex> eventLock(mutex);
                     Progressed(Now());
-                    NextTriggerChanged();
-                });
+                    NextTriggerChanged(); });
```

Comment:
The closing brace should be on a new line

Reply:
Setting the ColumnLimit fixes this; but with the expense of having deriving classes with a single base on a single line.

```
class Derived : public Base
{}
```